esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,884 @@
+          APHRO_V1101EX_R1/APHRO_MA
+              025deg_nc/APHRO_MA_025deg_V1101_EXR1.nc.tgz
+              050deg_nc/APHRO_MA_050deg_V1101_EXR1.nc.tgz
+
+      Please untar / unzip all *.tar *.tgz *.gz files in the same directory
+      (no subdirectories!) prior to running the cmorizer!
+
+  AURA-TES:
+    tier: 3
+    source: https://search.earthdata.nasa.gov/search?q=TL3O3M
+    last_access: 2018-12-08
+    info: |
+      Download and processing instructions
+      Select the V004 and V005 projects.
+      Download the script file for both projects.
+      To download the data use: wget -i <scriptfile>.
+      From the downloaded data, pick only the *.he5 files and put them in
+      input_dir_path.
+      Data is freely available, but a registration is required.
+
+  BerkeleyEarth:
+    tier: 2
+    source: http://berkeleyearth.org/data/
+    last_access: 2020-02-25
+    info: |
+      Download the following file:
+      http://berkeleyearth.lbl.gov/auto/Global/Gridded/Land_and_Ocean_LatLong1.nc
+
+  CALIPSO-GOCCP:
+    tier: 2
+    source: ftp://ftp.climserv.ipsl.polytechnique.fr/
+    last_access: 2020-01-27
+    info: |
+      Download the data from:"
+        ftp://ftp.climserv.ipsl.polytechnique.fr/
+        path: /cfmip/GOCCP_v3/3D_CloudFraction/grid_2x2xL40/yyyy/avg/
+      Put all files under a single directory (no subdirectories with years).
+        3D_CloudFraction330m_yyyymm_avg_CFMIP2_sat_3.1.2.nc
+
+  CDS-SATELLITE-ALBEDO:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-albedo?tab=form
+    last_access: 2019-04-01
+    info: |
+      Download and processing instructions
+        - Download the data from source to the right directory
+          using the download script
+        - Decompress the files within the directory:
+            "find . -name '*.tar.gz' -execdir tar -xzvf '{}' \"
+
+  CDS-SATELLITE-LAI-FAPAR:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-lai-fapar?tab=form
+    last_access: 2019-07-03
+    info: |
+      Download and processing instructions
+      - Open in a browser the data source as specified above
+      - Put the right ticks:
+        - Tick variables LAI and FAPAR
+        - Tick satellite SPOT (System Pour l'Observation de la Terre)
+        - Tick sensor VGT (Vegetation)
+        - Tick horizontal resolution 1km
+        - Tick product version V1
+        - Tick all available years
+        - Tick all available months
+        - Tick Nominal day 20
+      - Click 'submit form'
+      - According to ESMValTool practice, put them in the right rawobsdir folder
+      - Request might need to be split into chunks to not exceed download limit
+      - Fails setting standard name for variable FAPAR
+
+  CDS-SATELLITE-SOIL-MOISTURE:
+    tier: 3
+    source: https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-soil-moisture?tab=form
+    last_access: 2019-03-14
+    info: ""
```

Comment:
For the CDS-* datasets, the automatic download procedure only works if the user stores the cds credentials in the `~/.cdsapirc` file. Shall we add some documentation on how to generate this file? This [link ](https://cds.climate.copernicus.eu/api-how-to) could be enough.

The user is warned if the `.cdsapirc` file is missing:
```
Exception: Missing/incomplete configuration file: ~/.cdsapirc
2021-09-17 13:58:16,021 UTC [15664] ERROR   Download failed for datasets ['CDS-SATELLITE-SOIL-MOISTURE']
```
but it may not seem clear how to generate one.

Reply:
Added the link to the error message when this happens.